File: C++/Linked List that is Sorted Alternatingly.cpp
class Solution
{
    public:
    void sort(Node **head)
    {
        for(Node* p=*head; p!= NULL ; p=p->next){
            for(Node* q =p->next; q != NULL; q =q->next){
                if(p->data > q->data){
                    swap(p->data, q->data);
                }
            }
        }
    }
};

File: JAVA/Linked List that is Sorted Alternatingly.java
class Solution {
   public Node sort(Node head){
        ArrayList<Integer> list = new ArrayList<>();
        for(Node temp = head; temp != null; temp = temp.next){
            list.add(temp.data);
        }
        Collections.sort(list);
        Node temp = head;
        for(int n : list){
            temp.data = n;
            temp = temp.next;
        }
        return head;
   }
}
